user_interactions/user_interaction_txt_text.py

```diff
@@ -0,0 +1,69 @@
+import os
+from datetime import datetime
+
+from user_interactions.base_user_interaction import (BaseUserInteraction,
+                                                     MessageScope)
+
+
+class UserInteractionFileManager():
+    def __init__(self, history_dir):
+        self.history_dir = history_dir
+        if not os.path.exists(self.history_dir):
+            os.mkdir(self.history_dir)
+
+        current_date = datetime.now().strftime('%Y-%m-%d-%H.%M.%S')
+        self.dir = os.path.join(self.history_dir, f"Game-{current_date}")
+        os.mkdir(self.dir)
+
+    def update_history_text(self, player_name, text):
+        self.file_dir = f"{self.dir}/{player_name}.txt"
+        with open(self.file_dir, mode='a') as file:
+            file.write(text + '\n')
+
+
+class UserInteraction(BaseUserInteraction):
+    def __init__(self, context: 'Context'):
+        super().__init__(context)
+        self.us_inter_file_manager = UserInteractionFileManager(
+            os.path.join(os.path.dirname(__file__), "History"))
```

Comment:
Maybe it's better to move os.path.join(os.path.dirname(__file__) logic to a UserInteractionFileManager?
And open file inside filemanager?

Reply:
Done